src/test/java/com/glovoapp/planningpoker/e2e/nametest/SameNameInSessionTest.java

```diff
@@ -0,0 +1,52 @@
+package com.glovoapp.planningpoker.e2e.nametest;
+
+import com.glovoapp.planningpoker.e2e.WebDriverFactory;
+import com.glovoapp.planningpoker.e2e.pages.MainPage;
+import com.glovoapp.planningpoker.e2e.pages.SessionPage;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.WebDriver;
+
+import static com.glovoapp.planningpoker.e2e.WebDriverFactory.getDriver;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class SameNameInSessionTest {
+
+    private final WebDriver driver = getDriver();
+    private final WebDriver driver2 = getDriver();
+    private MainPage firstUsersMainPage = new MainPage(driver);
+    private SessionPage firstUsersSessionPage = new SessionPage(driver);
+    private MainPage secondUsersMainPage = new MainPage(driver2);
+    private SessionPage secondUsersSessionPage = new SessionPage(driver2);
+
+    private final String sessionName = "test2";
+    private final String userName = "michal";
```

Comment:
This constant should be static. Also, please be consistent with naming; you sometimes use `userName` and other times `username`:

```suggestion
    private static final String USERNAME = "michal";
```